src/Symfony/Bundle/Resources/config/maker.php

```diff
@@ -17,14 +17,14 @@
     $services = $container->services();
 
     $services->set('api_platform.maker.command.state_processor', 'ApiPlatform\Symfony\Maker\MakeStateProcessor')
-        ->args([service('api_platform.metadata.resource.name_collection_factory')])
```

Comment:
`service('api_platform.metadata.resource.name_collection_factory')` was introduced in the original PR
But there was no dependency on this class then, or is now, so I'm assuming this was a leftover from en earlier approach